### PR TITLE
Revert bc.tls-fips version to 1.0.13

### PR DESCRIPTION
**Pull request by adirwill**
Fix cp release test failures.
reference [thread](https://confluent.slack.com/archives/C027DGB2EHG/p1731389874638499?thread_ts=1729769918.268309&cid=C027DGB2EHG)

### CODE REVIEW

**Comment by trnguyencflt** [comment]
Hi @adirwill would you mind updating the description explaining the reason why there is downgrade in the version, the slack thread is very long and it is quite hard to follow it.

**Comment by adirwill** [comment]
> Hi @adirwill would you mind updating the description explaining the reason why there is downgrade in the version, the slack thread is very long and it is quite hard to follow it.

Tagging @rrbadiani @ujjwalkalia for context with cp release

**Review by janjwerner-confluent — APPROVED** [review]
lgtm
